dandi/tests/fixtures.py

```diff
@@ -269,6 +269,14 @@ def fixture(tmp_path_factory: pytest.TempPathFactory) -> str:
         path = str(tmp_path_factory.mktemp("gitrepo"))
         lgr.debug("Cloning %r into %r", url, path)
         run(["git", "clone", "--depth=1", url, path], check=True)
+        for i in os.listdir(path):
+            bids_dataset_path = os.path.join(path, i)
+            if os.path.isdir(bids_dataset_path):
+                dandiset_metadata_file_path = os.path.join(
+                    bids_dataset_path, dandiset_metadata_file
+                )
+                with open(dandiset_metadata_file_path, "w") as f:
+                    f.write(" \n")
```

Comment:
this is a generic `get_gitrepo_fixture`. I really would prefer it to do exactly that and not anyhow change that cloned git repo unconditionally !  It feels like you want to create some tuned up fixture based on this, called it some descriptive name/docstring on what side effect it creates on top of a clone.  But also not clear to me what is the goal here -- make every somefolder into a dandiset there?

or may be add it as an option to this fixture, e.g. `make_subfolders_into_dandisets=False` (or whatever would fit the purpose here) and enable thiscode block conditionally on that.

Reply:
done in 6bfb52045dfcd5cef1e5ad0d681c439fce324056